application_service/app_service.mak

```diff
@@ -83,6 +83,7 @@ test_user.exe: $(user_dobj)
 	@echo "linking executable files"
 	$(LINK) -o $(EXE_DIR)/test_user.exe $(user_dobj) $(LDFLAGS)
 
+$(I)/certifier.pb.h: $(US)/certifier.pb.cc
```

Comment:
This is the pattern of fix you will see in many Makefiles. This ensures that we will run `protoc` only once and -j n make builds will now succeed.

Reply:
Noticed this once before. Thanks for the fixes.